docs/notebooks/intro.py

```diff
@@ -43,6 +43,7 @@ def motorcycle_data():
     df = pd.read_csv("./data/motor.csv", index_col=0)
     X, Y = df["times"].values.reshape(-1, 1), df["accel"].values.reshape(-1, 1)
     Y = (Y - Y.mean()) / Y.std()
+    X /= X.max()
```

Comment:
How is this relevant to the problem?

Reply:
Rescales inputs between [0,1] instead of the [0,60], which makes the length-scales initialisation to 1 more sensible.